hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -164,6 +188,33 @@ public static void init(Context context, String password, LogLevel logLevel) {
     }
 
     /**
+     * Gets the saved data, if it is null, default value will be returned
+     *
+     * @param key          is used to get the saved data
+     * @param defaultValue will be sent to onSuccess if the response is null
+     * @param callback callback's onSuccess or onFail methods will be executed when action is completed
+     */
+    public static <T> void get(final String key, final T defaultValue, final Callback<T> callback) {
+        new Thread(new Runnable() {
+            @Override
+            public void run() {
+                String fullText = storage.get(key);
+                try {
+                    T t = encoder.decode(fullText);
+                    if(t == null) {
```

Comment:
we can inline here with 
<pre>
callback.onSuccess(t != null ? t : defaultValue);
</pre>